mscore/uploadscoredialog.cpp

```diff
@@ -78,6 +79,7 @@ UploadScoreDialog::UploadScoreDialog(LoginManager* loginManager)
 
       tagsHelp->setText(tr("Use a comma to separate the tags"));
```

Comment:
This text is a duplicate, same on the UI file, so needs to get translated twice, I think